src/test/java/com/issoft/training/tests/AutomationPracticeShoppingTests.java

```diff
@@ -0,0 +1,188 @@
+package com.issoft.training.tests;
+
+import com.issoft.training.base.TestBase;
+import com.issoft.training.config.ConfProperties;
+import com.issoft.training.listener.AllureListener;
+import com.issoft.training.pages.*;
+import com.issoft.training.utility.Constants;
+import io.qameta.allure.*;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import static com.issoft.training.config.ConfProperties.getProperty;
+
+@Listeners({ AllureListener.class })
+@Epic("Regression tests")
+@Feature(value = "Automation practice - Shopping tests")
+public class AutomationPracticeShoppingTests extends TestBase {
```

Comment:
It is not good practice to make tests depends on each other. It make all tests less stable.
The best practice is to make independent tests that each test can be executed without need to know the state of the previous test. 
Please remove priority and write tests in the independent way

Reply:
Priorities has been removed.